modules/ecs_policy/instance_policy.tf

```diff
@@ -0,0 +1,79 @@
+data "aws_iam_policy_document" "container_perms" {
+  statement {
+    actions = [
+      "ssm:GetParameters",
+    ]
+
+    resources = [
+      "arn:aws:ssm:ap-southeast-2:538673716275:parameter*",
+    ]
+  }
+
+  statement {
+    actions = [
+      "kms:*",
+    ]
+
+    resources = [
+      "arn:aws:kms:ap-southeast-2:538673716275:key/efba9b4c-8e64-430c-86c9-f00eaf69e582",
```

Comment:
Unsure where this is created, but if you give the key an alias it can be referenced with https://www.terraform.io/docs/providers/aws/d/kms_alias.html instead of being hardcoded.